src/Lifecycle/Guards.php

```diff
@@ -52,7 +51,7 @@ public function validate(): static
         }
 
         if (method_exists($this->event, 'failedValidation')) {
-            $this->event->failedValidation($this->state);
```

Comment:
This is technically a breaking change but I don't think anyone uses this functionality yet so it should be fine.